AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -126,7 +127,7 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
                     gaussian = fitobj.fit(x.T,y,y_err ) 
                     #Fitted beam center is in (x, y) coordinates, in projection centred on target
                     snr = np.abs(np.r_[gaussian.std/gaussian.std_std])
-                    valid_fit = np.all(np.isfinite(np.r_[gaussian.mean,gaussian.std_mean,gaussian.std,gaussian.std_std,gaussian.height,gaussian.std_height,snr]))
+                    valid_fit = np.all(np.append(np.isfinite(np.r_[gaussian.mean,gaussian.std_mean,gaussian.std,gaussian.std_std,gaussian.height,gaussian.std_height,snr]),gaussian.height > 0.01))
```

Comment:
This seems simpler than the corresponding code further down. How about a helper function that takes in `gaussian` and determines its validity? I did a similar thing in `scape.beam_baseline.BeamPatternFit`.

Speaking of which, that validity check is:
  - not isnan(mean)
  - height > 0
  - 0.9 < width / expected_width < 1.25